packages/suite/src/support/messages.ts

```diff
@@ -3613,6 +3613,46 @@ export default defineMessages({
         defaultMessage:
             'Legacy uses simpler transaction formats but may result in higher transaction fees and lacks the efficiency and features found in newer address types.',
     },
+    TR_ACCOUNT_TYPE_LEDGER_DESC: {
+        id: 'TR_ACCOUNT_TYPE_LEDGER_DESC',
+        defaultMessage:
+            'Ledger accounts are compatible with Ledger Live derivation paths, enabling smooth  migration from Ledger to Trezor.',
+    },
+    TR_ACCOUNT_TYPE_LEGACY_DESC: {
+        id: 'TR_ACCOUNT_TYPE_LEGACY_DESC',
+        defaultMessage:
+            'Legacy accounts are compatible with Ledger Legacy derivation paths, enabling smooth migration from Ledger to Trezor.',
+    },
+    TR_ACCOUNT_TYPE_NORMAL_ETH_DESC: {
+        id: 'TR_ACCOUNT_TYPE_NORMAL_ETH_DESC',
+        defaultMessage:
+            'The current and most widely accepted method of generating and managing Ethereum addresses ensures interoperability, security, and support for all types of tokens.',
+    },
+    TR_ACCOUNT_TYPE_NORMAL_ETH_CLASSIC_DESC: {
+        id: 'TR_ACCOUNT_TYPE_NORMAL_ETH_CLASSIC_DESC',
+        defaultMessage:
+            'Ethereum Classic is a fork of Ethereum that maintains the original, unaltered blockchain, emphasizing immutability.',
+    },
+    TR_ACCOUNT_TYPE_NORMAL_POLYGON_DESC: {
+        id: 'TR_ACCOUNT_TYPE_NORMAL_POLYGON_DESC',
```

Comment:
Do it generic for evms, name of network can be added as a value

Reply:
I think that in this case, it will be cleaner to use hardcoded string ids without using dynamic values for the following reasons:

-The utility getAccountTypeDesc() that comes from suite-common/wallet-utils doesn't import hooks from suite hooks (to use either useTranslation()) or <Translation/> component and do all the dynamic logic inside the utility, instead of rewriting it multiple times where this utility is used. 

It returns just strings that are ids for translations (it was already used before like that, now I have expanded its functionality). 

To make it dynamic for EVM name values, I'd have to import one of the translation methods into wallet-utils, or create an additional function/utility in packages/suite, to handle the logic of providing the translationString to the value in the translation component when there's a EVM description. Which also feels like an overkill for rewriting one word in ~3 strings. 

Especially considering that the strings are pretty generic right now and in the future, we might want to rewrite them anyway to provide more information about each chain.

What do you think about it? Does it make sense why I'd prefer to keep them hardcoded?

Reply:
Okay so I figured how to make it pretty dynamic in the end, please check it